asciidoc/runtime/pom.xml

```diff
@@ -20,6 +20,11 @@
             <artifactId>asciidoc-java</artifactId>
             <version>${asciidoc.java.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.yupiik.maven</groupId>
+            <artifactId>ascii2svg</artifactId>
```

Comment:
The error in Roq was the SVG was not available at runtime ClassNotFound. My choice was add this depedenecy and make it available or do a GraalVM Substitute and remove the capability from AsciiDoc?  Up to you I chose the route of not taking away functionality rather adding it.

Reply:
If this dependency is needed by Roq then I don't think it makes sense to add it here? 

@ia3andy Am I missing something?

Reply:
@mkouba I think it's needed for asciidoc, Roq is not doing anything more fancy with it

Reply:
@mkouba what @melloware is saying I think is that without this it doesn't work in native (in qute-web) unless doing things to remove some class so he added it to make it work (instead of removing it)

Reply:
Well, in that case the asciidoc module is missing integration tests, right? If so then it's the first thing we need to fix IMO.

Reply:
Yes, that makes sense